darts/utils/historical_forecasts/utils.py

```diff
@@ -208,6 +210,28 @@ def _historical_forecasts_general_checks(model, series, kwargs):
                 logger,
             )
 
+    # check that fit/predict_kwargs contain only supported parameters
+    if n.fit_kwargs is not None:
+        fit_args = set(inspect.signature(model.fit).parameters)
+        wrong_args = set(n.fit_kwargs) - fit_args
+        if len(wrong_args) > 0:
+            raise_log(
```

Comment:
Should just raise a warning here and below to just tell the users which params will be ignored? Because the fit/predict wrappers will ignore the params which are not valid.
Could be nice to use the same hist_fc parameters for any model